examples/CustomCollectionView/Sample/MosaicCollectionLayoutDelegate.m

```diff
@@ -36,11 +36,13 @@ - (instancetype)initWithNumberOfColumns:(NSInteger)numberOfColumns headerHeight:
 
 - (ASScrollDirection)scrollableDirections
 {
+  ASDisplayNodeAssertMainThread();
```

Comment:
Are these and the below necessary? Seems a little strange to be in 'client' code? Leftover from testing?